developer-docs-site/docs/tutorials/your-first-nft-sdk.md

```diff
@@ -0,0 +1,376 @@
+```
+=== Addresses ===
+Alice: 0x9df0f527f3a0b445e4d5c320cfa269cdefafc7cd1ed17ffce4b3fd485b17aafb
+Bob: 0xfcc74af84dde26b0050dce35d6b3d11c60f5c8c58728ca3a0b11035942a0b1de
+
+=== Initial Balances ===
+Alice: 20000
+Bob: 20000
+
+=== Creating Collection and Token ===
+Alice's collection: {
+    "description": "Alice's simple collection",
+    "maximum": "18446744073709551615",
+    "mutability_config": {
+        "description": false,
+        "maximum": false,
+        "uri": false
+    },
+    "name": "Alice's",
+    "supply": "1",
+    "uri": "https://aptos.dev"
+}
+Alice's token balance: 1
+Alice's token data: {
+    "default_properties": {
+        "map": {
+            "data": []
+        }
+    },
+    "description": "Alice's simple token",
+    "largest_property_version": "0",
+    "maximum": "1",
+    "mutability_config": {
+        "description": false,
+        "maximum": false,
+        "properties": false,
+        "royalty": false,
+        "uri": false
+    },
+    "name": "Alice's first token",
+    "royalty": {
+        "payee_address": "0x9df0f527f3a0b445e4d5c320cfa269cdefafc7cd1ed17ffce4b3fd485b17aafb",
+        "royalty_points_denominator": "1000000",
+        "royalty_points_numerator": "0"
+    },
+    "supply": "1",
+    "uri": "https://aptos.dev/img/nyan.jpeg"
+}
+
+=== Transferring the token to Bob ===
+Alice's token balance: 0
+Bob's token balance: 1
+
+=== Transferring the token back to Alice using MultiAgent ===
+Alice's token balance: 1
+Bob's token balance: 0
+```
+
+This example demonstrates:
+
+* Initializing the REST and Faucet clients
+* The creation of two accounts: Alice and Bob
+* The funding and creation of Alice and Bob's accounts
+* The creation of a collection and a token using Alice's account
+* Alice offering a token and Bob claiming it
+* Bob unilaterally sending the token to Alice via a multiagent transaction
+
+## Step 4: The SDK in Depth
+
+### Step 4.1: Initializing the Clients
+
+In the first step, the example initializes both the REST and Faucet clients. The REST client interacts with the REST API, whereas the Faucet client is a devnet
+ service for creating and funding accounts.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+```python
+:!: static/sdks/python/examples/simple-nft.py section_1
+```
+
+`common.py` initializes these values as such:
+```python
+:!: static/sdks/python/examples/common.py section_1
+```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+
+In progress
+  </TabItem>
+  <TabItem value="typescript" label="Typescript">
+
+In progress
+  </TabItem>
+</Tabs>
+
+:::tip
+
+The URLs for both services, by default, point to our devnet services, however, they can be configured via setting following environmental variables: `APTOS_NOD
+E_URL` and `APTOS_FAUCET_URL`.
+
+:::
+
+
+### Step 4.2: Creating local accounts
+
+The next step is to create two accounts from the locally. [Accounts][account_basics] represent both on and off-chain state. The off-chain state consists of an
```

Comment:
lol the `locally`

`The next step is to create two accounts locally.`

I wonder where we landed on `public, private` vs `public-private` vs `asymmetric` keypair.